spec/beaker/dsl/helpers/web_helpers_spec.rb

```diff
@@ -36,21 +36,43 @@ def fetch_allows
       it "returns its second and third arguments concatenated." do
         concat_path = "#{destdir}/#{name}"
         create_files([concat_path])
-        expect( logger ).to receive( :notify ).with( /^Already\ fetched\ / )
```

Comment:
Yeah, I expected the `create_files` method here to touch the file and then cause the logger to receive that `:notify`, so it was a little bit confusing, but I agree that we don't need the expect to be that specific here.